### PR TITLE
tau: update to 0.9.2; renamed from gxi.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by CameronNemo** [comment]
This is not really a rename, it drops gxi hard. Is that intentional? `pantheon-terminal` e.g. has a dummy subpackage, which is an alternative.

**Comment by Cogitri** [comment]
Oh alrighty, just looked at how it had been done for gnome-mpv

**Comment by Cogitri** [comment]
Thanks for the pointer, pushed new version

File: srcpkgs/tau/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+pkgname=tau
+version=0.9.2
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo pkg-config rust glib-devel"
+makedepends="gtk+3-devel rust-std"
+checkdepends="appstream-glib desktop-file-utils"
+short_desc="GTK frontend for the Xi text editor, written in Rust"
+maintainer="Enno Boland <gottox@voidlinux.org>"
```
Comment (Vaelatern) [inline srcpkgs/tau/template]: Why are you not the maintainer?

@Gottox weigh in, I suppose GXI was your package so you may have more knowledge about this package.

Comment (Cogitri) [reply srcpkgs/tau/template]: Ah, I guess thie was changed when Gottox adopted all of my packages. I can take maintainership of this again if you want to.

Comment (Vaelatern) [reply srcpkgs/tau/template]: Why did that initial adoption take place?

Comment (Cogitri) [reply srcpkgs/tau/template]: Because I'm not really active on Void anymore, so Gottox asked if he can take over my packages which was OK with me

**Comment by Cogitri** [comment]
Ah, forgot to push the fix for the checksum, will push in a bit.

File: srcpkgs/tau/template (hunk 1 above)
Comment (Vaelatern) [reply srcpkgs/tau/template]: then I'm OK with this maintainership.

**Pull request merged**